ygen/genstate.go

```diff
@@ -67,6 +70,16 @@ type MappedType struct {
 	DefaultValue *string
 }
 
+// MappedUnionSubtype stores information associated with a union subtype within
+// a MappedType.
+type MappedUnionSubtype struct {
+	// EnumeratedYANGTypeKey stores a globally-unique key that can be
+	// used to key into IR's EnumeratedYANGTypes map containing all of the
+	// enumeration definitions. This value should only be populated when
+	// the union subtype is an enumerated type
+	//EnumeratedYANGTypeKey string
```

Comment:
Is this commented out to avoid an unused warning?

Reply:
This is commented out since it's not being used so I wanted to make this clear. This is uncommented in the next PR: https://github.com/openconfig/ygot/pull/671